tendrl/monitoring_integration/grafana/alert_utils.py

```diff
@@ -125,7 +125,7 @@ def remove_cluster_rows(cluster_id, dashboard_name):
 
 
 def delete_panel(
-    integration_id, resource_type, resource_name=None
+    integration_id, resource_type=None, resource_name=None
```

Comment:
Why is resource_type optional now?

Reply:
In case we want to delete all the resource of a cluster we need only integration_id to be passed.

Reply:
This is used in un-manage cluster flow this way